CONTRIBUTING.rst

```diff
@@ -211,7 +211,8 @@ If your tutorial includes large data files (where large means >~ 1 MB), we don't
 want to include them in the ``astropy/astropy-tutorials`` git repository, as
 that will drastically slow down cloning the repository. Instead, we encourage
 use of the `astropy.utils.download_files` function, and will host data files on
-the `<http://data.astropy.org>`_ server. To do this, use the following
+the `<http://data.astropy.org>`_ server, or to use Zenodo to host the data files.
+To do the former, use the following
 procedure:
 
```

Comment:
Change this wording so that "We encourage writers to host data files on Zenodo, using the following procedure."

....

"In the past, we have used the `astropy.utils.download_files` function in conjunction with storing files on the `<http://data.astropy.org>`_ server. This requires the following procedure:"